etcdutl/snapshot/v3_snapshot.go

```diff
@@ -203,6 +204,16 @@ type RestoreConfig struct {
 	// SkipHashCheck is "true" to ignore snapshot integrity hash value
 	// (required if copied from data directory).
 	SkipHashCheck bool
+
+	// RevisionBump is the amount to increase the latest revision after restore,
+	// to ensure that the revisions never decrease from the clients perspective
+	// If 0, revision bumping is skipped.
+	RevisionBump int64
+
+	// MarkCompacted is "true" to mark the latest revision as scheduled for compaction.
+	// This will cause the server to compact the restored revisions (at some point after startup),
```

Comment:
```suggestion
	// This will cause the server to compact the restored revisions (at some point after startup),
```
This is implementation detail.